Examples/Swift/ViewController.swift

```diff
@@ -157,13 +157,12 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
                 print(error!.localizedDescription)
                 return
             }
-            guard let route = routes?.first else { return }
+            guard let routes = routes else { return }
```

Comment:
If `routes` is an empty array, should we still continue past this point?

Reply:
No, and we don't(?).